lib/node_modules/@stdlib/math/base/special/expm1rel/lib/main.js

```diff
@@ -26,10 +38,16 @@
 *
 * @example
 * var v = expm1rel( 0.0 );
-* // returns TODO
+* // returns 1.0
 */
 function expm1rel( x ) {
-	// TODO: implementation
+	if ( abs( x ) <= EPS) { // L'Hopitals rule
```

Comment:
Nit: L'Hopital's rule.

Reply:
I added some context to L'Hopitals rule too.

Reply:
Forget about this last comment, I just fixed the typo.